tests/automation-framework/rancherclient/config.go

```diff
@@ -0,0 +1,94 @@
+package rancherclient
+
+type Config struct {
+	CattleTestURL     string `json:"cattleTestURL"`
```

Comment:
can we rename this to `RancherHost`, url implies schemes, ports, paths.

Reply:
Done